schemas/dex-aggregator/SCHEMA.md

```diff
@@ -22,36 +23,37 @@ Please follow the below schemas for integrating a DEX Aggregator with OpenBlock
-| incentiveAmount | The cumulative amount of incentives.                            | number |
-| incentiveUsd    | The value of incentives in USD.                                 | number |
+| dailyVolumeAmount    | The daily volume amount of the token (in native, decimal normalized terms). | number |
+| dailyVolumeUsd       | The daily volume of the token in USD.                      | number |
+| dailyFees         | The daily fees taken from this token in native, decimal normalized terms. | number |
+| dailyFeesUsd         | The daily fees collected in USD.                           | number |
+| dailyIncentiveAmount | The daily amount of incentives collected from this token.  | number |
+| dailyIncentiveUsd    | The daily value of incentives in USD.                      | number |
 
 ### Protocol Snapshot
 
 | Property     | Description                                             | Type   |
 |--------------|---------------------------------------------------------|--------|
-| timestamp    | The timestamp of the record.                            | number |
-| protocolName | The name of the protocol this table belongs to.         | string |
-| volumeUsd    | The cumulative volume in USD.                           | number |
-| feesUsd      | The cumulative fees collected in USD.                   | number |
+| timestamp    | The timestamp of the block this snapshot was taken.     | number |
+| protocolName | The name of the protocol.                               | string |
+| chainId      | The standard id of the chain.                           | number |
+| dailyVolumeUsd | The daily volume in USD.                              | number |
+| dailyFeesUsd   | The daily fees collected in USD.                      | number |
 
 ### Trades
 
 | Property           | Description                                                                 | Type       |
 |--------------------|-----------------------------------------------------------------------------|------------|
 | timestamp          | The timestamp of the trade.                                                 | number     |
 | protocolName       | The name of the protocol this table belongs to.                             | string     |
-| chainId            | The standard id of the chain this trade belongs to                          | number     |
+| chainId            | The standard id of the chain this trade belongs to.                         | number     |
 | blockNumber        | The block number in which the trade occurred.                               | number     |
 | transactionId      | The transaction id associated with this trade.                              | string     |
-| fees               | The amount of fees (ideally in USD terms) for this trade.                   | number     |
+| fees               | The amount of fees for this trade.                                          | number     |
```

Comment:
Are fees no longer planned to be in USD?

Reply:
added that in, thansk